src/server/src/main/docker/entrypoint.sh

```diff
@@ -2,9 +2,9 @@
 
 if [ "$1" = 'cassandra-reaper' ]; then
     set -x
-    su-exec reaper /usr/local/bin/append-persistence.sh
-    exec su-exec reaper java -jar ${JAVA_OPTS} \
-        /usr/local/lib/cassandra-reaper.jar server /etc/cassandra-reaper.yml
+    su-exec reaper /usr/local/bin/configure-persistence.sh
+    su-exec reaper /usr/local/bin/configure-metrics.sh
+    exec su-exec reaper java ${JAVA_OPTS} -jar /usr/local/lib/cassandra-reaper.jar server /etc/cassandra-reaper.yml
```

Comment:
Mick mentioned wanting to use the `bin/start` file. Might want to be updated here.

Also a `\ ` newline may be good to highlight the different arguments.

Reply:
If it is ok, can we leave off the change use the `bin/start` file here please. Reason being is that file will need some changes to allow addition `JAVA_OPTS` to be passed to it.  We should raise a separate PR to make that change.

In the meantime, the suggestion to add a `\` newline to highlight the different arguments has been applied.